source/text/implementation/vss-string_vectors.adb

```diff
@@ -98,6 +98,25 @@ package body VSS.String_Vectors is
       VSS.Implementation.String_Vectors.Unreference (Self.Data);
    end Clear;
 
+   --------------
+   -- Contains --
+   --------------
+
+   function Contains
+     (Self : Virtual_String_Vector'Class;
+      Item : VSS.Strings.Virtual_String'Class) return Boolean
+   is
+      use VSS.Strings;
+   begin
+      for Element of Self loop
+         if Element = Virtual_String (Item) then
```

Comment:
Almost same here - it uses high level API. 

Also, string handler of the Item can be resolved once and used to check equality with each string in the list.